modules/ROOT/pages/design-common-problems-raml-10.adoc

```diff
@@ -284,10 +288,56 @@ Name:
       a: "b"
       b: 1
 ----
+////
+
+=== Naming examples twice
+
+If an example is named twice, the editor displays this message, where `<name-of-property>` is the name of the property that the named example is an example of:
+
+----
+should have required property '<name-of-property>'
+----
+
+For example, suppose that a specification defines the property `firstName` and links to a NamedExample fragment by giving the example the name `someName`:
+----
+#%RAML 1.0
+title: Example API Spec
+/tiers:
+ post:
+   body:
+     application/json:
+       properties:
+         firstName: string
+       examples:
+         someName: !include ex.raml
+----
+
+Further suppose that the NamedExample fragment also names the example, giving it the name `someFragmentName`:
+----
+#%RAML 1.0 NamedExample
+someFragmentName:
+  firstName: Martin Gutierrez
+----
+
+In this case, the editor would display this message because the parser interprets `someFragmentName` as the name of the property:
+
+----
+should have required property 'firstName'
+----
+
+To resolve this type of problem, remove either the name in the `examples` facet in the specification or the name in the NamedExample fragment.
+
+If you choose to remove the name from the NamedExample fragment, you must remove the word "NamedExample" from the declaration in the fragment. For example, you would change the NamedExample fragment used in the situation above to look like this:
+
+----
+#%RAML 1.0
+firstName: Martin Gutierrez
+----
+
 
 
 ////
-//=== Including more than one named example in a single NamedExample fragment
+=== Including more than one named example in a single NamedExample fragment
```

Comment:
@kbostrom , this section is still commented out. I removed the unnecessary forward slashes because the four forward slashes in the previous line override them.